atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/api/TestTimestampCommand.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.cli.api;
+
+import java.util.Scanner;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.palantir.atlasdb.cli.TimestampCommand;
+import com.palantir.atlasdb.transaction.api.TransactionTask;
+import com.palantir.atlasdb.transaction.impl.RawTransaction;
+import com.palantir.lock.LockRefreshToken;
+
+import io.airlift.airline.Cli;
+
+public class TestTimestampCommand {
+
+    private static String configPath;
+    private static Cli<SingleBackendCommand> cli;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        configPath = SingleBackendCliTests.getConfigPath(SingleBackendCliTests.SIMPLE_ROCKSDB_CONFIG_FILENAME);
+        cli = SingleBackendCliTests.build(TimestampCommand.class);
+    }
+
+    @Test
+    public void testBasicInvariants() throws Exception {
+        SingleBackendCommand cmd = cli.parse("timestamp", "-c", configPath, "-f", "-i");
+        AtlasDbServices services = cmd.connect();
```

Comment:
will we have to do some kind of cleanup on our services after running the test? thinking whether this could end up leaving a bunch of thread pools/connections/etc around when we have multiple tests like this. if we do need to do cleanup, might also make sense to make it part of SingleBackendCommand.